druid/src/widget/image.rs

```diff
@@ -25,7 +25,52 @@ use crate::{
     PaintCtx, Rect, RenderContext, Size, UpdateCtx, Widget,
 };
 
-/// A widget that renders an Image
+/// A widget that renders a bitmap Image.
+///
+/// Contains data about how to fill given space and interpolate pixels.
+/// Configuration options are provided via the builder pattern.
+///
+/// Note: interpolation can lead to blurry or pixelated images and so is not
+/// recommended for things like icons. Instead consider using
+/// [SVG files](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics)
+/// and enabling the `svg` feature with `cargo`.
+///
+/// (See also:
+/// [`ImageData`],
+/// [`FillStrat`],
+/// [`InterpolationMode`]
```

Comment:
These links aren't defined?

Reply:
they don't have to be when `rustdoc` can figure them out (i haven't looked into the scope of this in detail). these links work as expected, AFAICT, but it's not problem if they need to be updated to match the more bibliographical approach in the other docs.

Reply:
The links certainly don't work with stable rust. I know that intra-doc rustdoc links are [on their way to stabilization](https://github.com/rust-lang/rust/issues/43466) but they aren't here yet. Our docs need to work with stable rust.

Reply:
ah, that's the missing piece. sorry about that.